### PR TITLE
Features/initial outline 176782628

### PR DESCRIPTION
**Pull request by rosalinekarr**
Problem
=======
We need a basic outline to structure our thoughts and give us a jumping off point.
[#176782628](https://www.pivotaltracker.com/story/show/176782628)

Solution
========
Add a bunch of files and directories with some basic outline of the content

### CODE REVIEW

File: pages/DSNP/Messages/Overview.md [hunk 1]
```diff
@@ -18,7 +23,7 @@
 ## Assumptions
 * Chain messages are on Ethereum
 * Message data is posted via [Ethereum log events](https://medium.com/mycrypto/understanding-event-logs-on-the-ethereum-blockchain-f4ae7ba50378)
-* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use `ecreover` 
+* Signature algorithm is [secp256k1](https://en.bitcoin.it/wiki/Secp256k1). This allows the use `ecreover`
```
Comment (shannonwells) [inline pages/DSNP/Messages/Overview.md:26]: allows the use _of_ `ecrecover`, my bad

File: pages/DSNP/Messages/Serialization.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
-# DSNP Message Serialization
+---
+name: Message Serialization
```
Comment (shannonwells) [inline pages/DSNP/Messages/Serialization.md:2]: 👍  good to know, I missed this

File: pages/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+These challenges include censorship from state and corporate actors, the amplification of misinformation through viral content, and an ever-shrinking collection of near monopolies with absolute power over social interaction in the 21st century.
+Through the Liberty Protocol, we hope mitigate and ideally solve these challenges in the way social interaction operates online.
+
+## Censorship
+
+TBD - Quick summary on the threat of censorship online; maybe Harry has some suggestions
+
+## Misinformation
+
+TBD - Quick summary on the threat of misinformation; ask Shannon for input
```
Comment (shannonwells) [inline pages/index.md:23]: I really think these items don't belong in a spec, which is aimed at helping developers use the platform.  At most they should link to outside articles, such as the blog site.

**Review by shannonwells — APPROVED** [review]
Generally looks great. I'm fine with leaving the non-development related content as TBD for now.

File: pages/index.md (hunk 3 above)
Comment (rosalinekarr) [reply pages/index.md:23]: I'm just going on the feedback from the OTS meeting. I think the general consensus was that we wanted to briefly mention this stuff here to drive home the point, but the bulk of this writing will be on the developer portal.

Comment (shannonwells) [reply pages/index.md:23]: Ah! Ok. I either missed that entirely or just forgot about it.

**Pull request merged**